metrologist/field_illum/views.py

```diff
@@ -0,0 +1,60 @@
+"""
+homogeneity report elements to include inside templates/homogeneity.html
+missing:
+- specify path to homogeneity_module from MetroloJ-for-python
+- path_temp: define path to save-in and load-from png files.
+"""
+
+import os
+import requests
+import tempfile
+import logging
+from datetime import datetime
+
+from flask import (
+    Blueprint,
+    render_template,
+    request,
+    redirect,
+    flash,
+    url_for,
+    send_file,
+    current_app,
+)
+
+from flask_login import login_required, current_use
+
+from metrologist.field_illum.models import Homogeneity
+
+import code.homogeneity_module as homo
+
+
```

Comment:
here you need to add a mechanism where you upload the image data and perform the computattion